docs/application/dotnet/guides/nui/nui-components/Switch.md

```diff
@@ -0,0 +1,65 @@
+# Switch
+This tutorial describes how to create and use Popup.
+
+## Overview
+Switch is one kind of common component, it can be used as selector.
+
+## Create with property
+1. Create Switch by default constructor
+
+~~~{.cs}
+Switch[] utilitySwitch = new Switch[4];
+~~~
+
+2. Set switch property
+
+~~~{.cs}
+int num = 4;
+for(int i = 0; i < num; i++)
+{
+    utilitySwitch[i] = new Switch();
+    utilitySwitch[i].Size = new Size(96, 60);
+    utilitySwitch[i].Position = new Position(300 + 100 * i, 300);
+    utilitySwitch[i].SwitchHandlerImageSize = new Size(60, 60);
+    utilitySwitch[i].SwitchBackgroundImageURLSelector = new StringSelector
+    {
+        Normal = "controller_switch_bg_off.png",
+        Selected = "controller_switch_bg_on.png",
+        Disabled = "controller_switch_bg_off_dim.png",
+        DisabledSelected = "controller_switch_bg_on_dim.png",
+    };
+    utilitySwitch[i].SwitchHandlerImageURLSelector = new StringSelector
+    {
+        Normal = "controller_switch_handler.png",
+        Selected = "controller_switch_handler.png",
+        Disabled = "controller_switch_handler_dim.png",
+        DisabledSelected = "controller_switch_handler_dim.png",
+    };
+    root.Add(utilitySwitch[i]);
+}
+utilitySwitch[2].IsEnabled = false;
+utilitySwitch[2].IsSelected = false;
+utilitySwitch[3].IsEnabled = false;
+utilitySwitch[3].IsSelected = true;
+~~~

```

Comment:
The enabled/selected setting values for each switch are not matched with the result image `media/SwitchProperty.PNG`

Reply:
Done